tools/run-tests.py

```diff
@@ -455,16 +464,19 @@ def main(options):
         Check(options.check_format, run_check, [settings.FORMAT_SCRIPT]),
         Check(options.check_license, run_check, [settings.LICENSE_SCRIPT]),
         Check(options.check_strings, run_check, [settings.STRINGS_SCRIPT]),
-        Check(options.jerry_debugger, run_jerry_debugger_tests, options),
-        Check(options.jerry_tests, run_jerry_tests, options),
-        Check(options.test262, run_test262_test_suite, options),
-        Check(options.unittests, run_unittests, options),
-        Check(options.buildoption_test, run_buildoption_test, options),
+        Check(options.jerry_debugger, run_jerry_debugger_tests, None),
+        Check(options.jerry_tests, run_jerry_tests, None),
+        Check(options.test262, run_test262_test_suite, None),
+        Check(options.unittests, run_unittests, None),
+        Check(options.buildoption_test, run_buildoption_test, None),
     ]
 
     for check in checks:
         if check.enabled or options.all:
-            ret = check.runner(check.arg)
+            if check.arg is None:
```

Comment:
Could you clarify your reasoning? It seems this might be a bit more complex than necessary.